CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+# 0.2.3 - 2019-11-11
+
+## Added
+
+ - Preliminary support for 3.8 walrus operator.
+ - CI config and fuzz tests for 3.8.
+ - Experimental re-entrant codegen API.
+ - Added `unsafe_skip_copy` optimization to `MetadataWrapper`.
+ - Matchers API now includes a `findall` function.
+ - Matchers now have a `MatchMetadataIfTrue` special matcher.
+
+## Updated
+
+ - Updated to latest Black release.
+ - Better type documentation for generated matchers.
+
+## Fixed
+
+ - Clarified matchers documentation in several confusing areas.
+ - Drastically sped up codegen and tests.
+ - `QualifiedName` now supports imported attributtes.
+ - `ExpressionContext` properly marks loop variables as `STORE`.
+ - Various typos in documentation are fixed.
+
+## Deprecated
+
+ - Deprecated `BasicPositionProvider` and `SyntacticPositionProvider` in favor of `WhitespaceInclusivePositionProvider` and `PositionProvider`.
```

Comment:
This was included in previous release.

Reply:
It looks like the  commit landed after the 0.2.2 bump, so I'll fix the changelog accordingly.